autokeras/hypermodels/preprocessing.py

```diff
@@ -121,26 +122,71 @@ class ImageAugmentation(block_module.Block):
     """
 
     def __init__(self,
-                 translation=True,
-                 horizontal_flip=True,
-                 vertical_flip=True,
+                 translation_range=0.5,
```

Comment:
About `translation_range` and `rotation_range`: they aren't quite ranges (a range implies two values, e.g. `[-0.5, 0.5]`). We call this argument `factor` in `RandomRotation` (and `height/width_factor` in RandomTranslation). `rotation_factor`, `translation_factor` would seem appropriate here. What do you think?

Reply:
Yes, those better names. I will rename them.